app/views/layouts/application.html.erb

```diff
@@ -13,8 +13,12 @@
   <body>
     <%= render "layouts/header" %>
     <div class="container">
+      <% flash.each do |message_type, message| %>
+          <%= content_tag(:div, message, class: "alert alert-#{message_type}") %>
```

Comment:
indent

Reply:
chuỗi k có kí tự in hoa, thì nó sẽ giữ nguyên ạ